### ISSUE TITLE
Saving Images with bounding boxes, not on windows!?

### ISSUE THREAD

**Issue by seanbenhur** [ultralytics/yolov5#4491]
## Additional context
I have trained a model and while testing I was able to save images with bounding boxes on colab, but I can't do the same on my local windows, the images stored on `runs\detect\exp` don't contain any boxes, Am I missing out on anything!?

`@torch.no_grad()
def run(weights='best.pt',  # model.pt path(s)
        source='data/images',  # file/dir/URL/glob, 0 for webcam
        imgsz=640,  # inference size (pixels)
        conf_thres=0.25,  # confidence threshold
        iou_thres=0.45,  # NMS IOU threshold
        max_det=1000,  # maximum detections per image
        device='cpu',  # cuda device, i.e. 0 or 0,1,2,3 or cpu
        view_img=False,  # show results
        save_txt=False,  # save results to *.txt
        save_conf=False,  # save confidences in --save-txt labels
        save_crop=True,  # save cropped prediction boxes
        nosave=False,  # do not save images/videos
        classes=None,  # filter by class: --class 0, or --class 0 2 3
        agnostic_nms=False,  # class-agnostic NMS
        augment=False,  # augmented inference
        visualize=False,  # visualize features
        update=False,  # update all models
        project='runs/detect',  # save results to project/name
        name='exp',  # save results to project/name
        exist_ok=False,  # existing project/name ok, do not increment
        line_thickness=3,  # bounding box thickness (pixels)
        hide_labels=True,  # hide labels
        hide_conf=True,  # hide confidences
        half=False,  # use FP16 half-precision inference
        tfl_int8=False,  # INT8 quantized TFLite model
        ):`

These are my function arguments

**Comment by wudashuo** [comment]
There's a bug on Windows, try to downgrade your CUDA to 10.2, and reinstall the corresponding version of PyTorch.

**Comment by seanbenhur** [comment]
Thanks, I don't have GPU, so no CUDA,, I will try downgrading pytorch, thanks
